pom.xml

```diff
@@ -1238,6 +1238,66 @@
         </plugins>
       </build>
     </profile>
+    <profile>
+      <id>yarn-with-corepack</id>
+      <activation>
+        <file>
+          <exists>.mvn_exec_yarn_corepack</exists>
+        </file>
+      </activation>
+      <build>
+        <plugins>
+          <plugin>
+            <groupId>com.github.eirslett</groupId>
+            <artifactId>frontend-maven-plugin</artifactId>
+            <version>${frontend-version}</version>
+
+            <executions>
+              <execution>
+                <id>install node and corepack</id>
+                <goals>
+                  <goal>install-node-and-corepack</goal>
+                </goals>
+                <configuration>
+                  <nodeVersion>v${node.version}</nodeVersion>
+                  <nodeDownloadRoot>${nodeDownloadRoot}</nodeDownloadRoot>
+                </configuration>
+              </execution>
+              <execution>
+                <id>yarn install</id>
+                <goals>
+                  <goal>corepack</goal>
+                </goals>
+                <configuration>
+                  <arguments>yarn install</arguments>
+                </configuration>
+              </execution>
+              <execution>
+                <id>yarn mvnbuild</id>
+                <goals>
+                  <goal>corepack</goal>
+                </goals>
+                <configuration>
+                  <arguments>yarn mvnbuild</arguments>
+                </configuration>
+              </execution>
+              <execution>
+                <id>yarn mvntest</id>
+                <goals>
+                  <goal>corepack</goal>
+                </goals>
+                <phase>test</phase>
+                <configuration>
+                  <arguments>yarn mvntest</arguments>
+                  <testFailureIgnore>${frontend.testFailureIgnore}</testFailureIgnore>
```

Comment:
I'm wondering whether its worth having this use default configuration, but then if someone doesn't configure a reporter their tests will silently fail in CI

Reply:
The way you have it now is consistent with the configuration elsewhere in this file, which is good.